server/src/main/java/io/druid/client/cache/CacheProvider.java

```diff
@@ -23,11 +23,12 @@
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
 import com.google.inject.Provider;
 
-@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type", defaultImpl = LocalCacheProvider.class)
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type", defaultImpl = CaffeineCacheProvider.class)
```

Comment:
Can you please edit the first comment of this PR to indicate that you do want to change the default? This threw me for a loop since it said you weren't going to, and I hadn't read the conversation between you and Ben yet where you changed your mind.

Reply:
Fixed